services/crypto_create.proto

```diff
@@ -128,8 +128,11 @@ message CryptoCreateTransactionBody {
     string memo = 13;
 
     /**
-     * The maximum number of tokens that an Account can be implicitly associated with. Defaults to 0
-     * and up to a maximum value of 1000.
+     * The maximum number of tokens that can be auto-associated with the account.<br/>
+     * If this is less than or equal to `used_auto_associations`, or 0, then this account
```

Comment:
Also, no need to note about `used_auto_associations` in `crypto_create.proto` because that is a value only in state and not in API imo

Reply:
It has an effect on an airdrop, however, so the thought is that noting it here can help a client developer understand what values affect how an airdrop transaction is expected to proceed.